lambda-buffers-frontend/resources/good/Test.lbf

```diff
@@ -0,0 +1,30 @@
+module Test
+
+import A
+
+import qualified B
+
+import qualified A.B
+import qualified A.B as AB
+
+import qualified C (A)
+import qualified C as D (A)
+import qualified C as D ()
+
+sum Maybe a = Just a | Nothing
+
+sum Either a b = Left a | Right b
+
+sum List a = Nil | List a (List a)
+
+opaque Int a
+opaque Bytes
+
+sum Foo a = MkFoo
+    A
+    A.A
+    B.A
+    A.B.A
+    AB.A
+    C.A
+    D.A
```

Comment:
For clarity reasons - I think the bar shouldn't be replaced by newline.
```suggestion
sum Foo a 
  = MkFoo
  |  A
  |  A.A
  |  B.A
  |  A.B.A
  |  AB.A
  |  C.A
  |  D.A
```

Reply:
The suggestion doesn't parse. This is a single constructor sum type that I just formatted weirdly to see if the parsing works (it does). Should be ideally formatted with `lambda-buffers-frontend-cli format`.